### ISSUE TITLE
What is "object_detection_ssd_mobilenetv2_oidv4_fp16.tflite" file for ?

### ISSUE THREAD

**Issue by dnap512** [google/mediapipe#1526]
While touring the TFLite models, I found the bject_detection_ssd_mobilenetv2_oidv4_fp16.tflite file.
[https://github.com/google/mediapipe/blob/master/mediapipe/models/object_detection_ssd_mobilenetv2_oidv4_fp16.tflite](https://github.com/google/mediapipe/blob/master/mediapipe/models/object_detection_ssd_mobilenetv2_oidv4_fp16.tflite)
It's not included in the description of meidiapipe models, could you tell me where to use this file?

**Comment by ahmadyan** [comment]
The Objectron model ([3D Object Detection](https://google.github.io/mediapipe/solutions/objectron)) is a two stage detector. The first stage detects a 2D object crop using the model you linked above, while the second stage estimates the 3D bounding box from the given crop.

![3D Object Detection](https://google.github.io/mediapipe/images/objectron_2stage_network_architecture.png)

You can refer to the [Tensorflow object detection tutorial](https://www.tensorflow.org/hub/tutorials/object_detection) for more information/examples.

**Comment by dnap512** [comment]
Thank you for reply.

**Comment by nargeshn** [comment]
I am trying to use object_detection_ssd_mobilenetv2_oidv4_fp16.tflite to get 2D object crops and use them as input to the second network but the results are not meaningful. 
Is the model trained and if yes what dataset is used for training?

**Comment by ahmadyan** [comment]
The SSD object detector is trained on JFT.

**Comment by nargeshn** [comment]
Thanks for your response!
I have two more questions and I would appreciate your response!
I use Tensorflow.image.combined_non_max_suppression on detected bounding boxes to get the normalized coordinates and the label of the detected class. Now my questions are:
1. All detected bounding boxes are classified as class one (label 0). Does it mean that this network is mainly used to get region proposals and does not return the class labels of detected boxes?
2. I tested the network on the simple images of one or two cups but most of the results after non-max suppression have either the width or height of zero or the bounding boxes are much larger or much smaller than the 2d ground truth bounding box. In other words, the detected bounding boxes don't match the ground truth bounding boxes. Is this behavior expected from this model?

**Comment by ahmadyan** [comment]
I recommend creating a new issue in mediapipe, as this issue is already closed.

The behavior you are describing does not sound normal. You can see the expected behavior of that graph in https://google.github.io/mediapipe/solutions/box_tracking and specifically in this graph for object detection with ssd:
https://github.com/google/mediapipe/blob/master/mediapipe/graphs/tracking/subgraphs/object_detection_gpu.pbtxt.

**Comment by nargeshn** [comment]
Thanks!
Sure, I'll create a new issue.
